internal/endtoend/testdata/emit_result_and_params_struct_pointers/query.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE foo (a integer, b integer);
+
+/* name: GetOne :one */
+SELECT * FROM foo WHERE a = ? LIMIT 1;
+
+/* name: GetAll :many */
+SELECT * FROM foo;
+
+/* name: GetAllAByB :many */
+SELECT a FROM foo WHERE b = ?;
```

Comment:
This does not test the case where the params struct should be a pointer to a struct.